src/templates/dandelion/config/screens/RedemptionsScreen.js

```diff
@@ -0,0 +1,244 @@
+      }
+    },
+    [redeemableTokens, next, dataKey, data]
+  )
+
+  return (
+    <form>
+      <Header
+        title={title}
+        subtitle={
+          <span
+            css={`
+              display: flex;
+              align-items: center;
+              justify-content: center;
+            `}
+          >
+            Choose your
+            <span
+              css={`
+                display: flex;
+                margin: 0 ${1.5 * GU}px;
+              `}
+            >
+              <KnownAppBadge
+                appName="redemptions.aragonpm.eth"
+                label={appLabel}
+              />
+            </span>
+            settings below.
+          </span>
+        }
+      />
+
+      <div
+        css={`
+          ${fieldsLayout};
+        `}
+      >
+        <Field
+          label={
+            <React.Fragment>
+              Redeemable tokens
+              <Help hint="What are redeemable tokens?">
+                <strong>Redeemable tokens</strong> are ERC20 tokens that can be
+                redeemed in exchange for organization tokens. This represent
+                assets within the Organization's vault.
+              </Help>
+            </React.Fragment>
+          }
+          css={`
+            margin: 0;
+          `}
+        />
+        <MultiTokenSelector
+          onAddToken={handleTokenAdded}
+          onRemoveToken={handleTokenRemoved}
+          onUpdateToken={handleTokenUpdated}
+          tokens={redeemableTokens}
+          items={DEFAULT_REDEEMABLE_TOKENS}
+        />
+      </div>
+
+      {formError && (
+        <Info
+          mode="error"
+          css={`
+            margin-bottom: ${3 * GU}px;
+          `}
+        >
+          {formError}
+        </Info>
+      )}
+
+      <Info
+        css={`
+          margin-bottom: ${3 * GU}px;
+        `}
+      >
+        These settings will determine which assets held within an organization’s
+        Vault are eligible for redemption.
+      </Info>
+
+      <Navigation
+        ref={prevNextRef}
+        backEnabled
+        nextEnabled
+        nextLabel={`Next: ${screens[screenIndex + 1][0]}`}
+        onBack={back}
+        onNext={handleSubmit}
+      />
+    </form>
+  )
+}
+
+RedemptionsScreen.propTypes = {
+  appLabel: PropTypes.string,
+  dataKey: PropTypes.string,
+  screenProps: ScreenPropsType.isRequired,
+  title: PropTypes.string,
+}
+
+RedemptionsScreen.defaultProps = {
+  appLabel: 'Redemptions',
+  dataKey: 'redemptions',
+  title: 'Configure template',
+}
+
+function formatReviewFields(screenData) {
+  return [
+    [
+      'Redeemable tokens',
+      <div>
+        {screenData.redeemableTokens.map(({ token }, index) => (
+          <div
+            key={index}
+            css={`
+              display: grid;
+              grid-template-columns: 1fr 2fr;
+            `}
+          >
+            <span>{token.symbol || 'Custom token'}</span>
+            <span> {shortenAddress(token.address)}</span>
```

Comment:
Here and in the `TokenRequestScreen`, we should detect `0x0000..0000` and ignore it.